.github/workflows/dev-long-tests.yml

```diff
@@ -69,6 +76,15 @@ jobs:
         make gcc8install
         CC=gcc-8 make -j uasan-test-zstd </dev/null V=1
 
+  clang-asan-ubsan-testzstd:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: clang + ASan + UBSan + Test Zstd
+      run: |
+        sudo apt-get -qqq update
```

Comment:
I don't think you need the `apt-get` command. It's used in the gcc-8 test because gcc8 is installed via apt by `make gcc8install`. If you are using the clang that's already installed in the CI environment, then I think you're good to delete the `apt-get`.

However, you should check the CI logs and make sure that clang is indeed present without requiring installation.